server/views/applications/index.njk

```diff
@@ -62,7 +62,8 @@
                   },
                   {
                     text: "Date started"
-                  }
+                  },
+                  {}
```

Comment:
Is there a way we can add some text here that doesn't render but explains what it is to screen readers?

I'm guessing we don't have a way of sending in the following?

```
<div aria-label="test A"></div>
```

I wonder what the axe plugin says but I wonder if having the text "Actions" would be reasonable.

Reply:
ah I see what you mean, will take a quick look

Reply:
So the govuk component has the ability to [add `html` within the `th` element](https://design-system.service.gov.uk/components/table/#options-table-example--head), but not to add a role or label to the actual `th` element. I think from [looking at accessibility guidelines](https://github.com/dequelabs/axe-core/issues/3404) that if we have a tableheader/columnheader then it should have some kind of name to it as thats semantically what it is there for... I'm going to err on the side of having a visible 'Actions' text within the `th` as it may not be exactly what the visual design was but I think it will make it more accessible.